vdm-io/Joomla-Component-Builder
# Minor improvments and bug fixes

Pull Request for Issue gh-285, gh-283, gh-278.

### Summary of Changes
- Decoupling JCB even more from vdm.io in preparation of the launch of the joomlacomponentbuilder.com website
- Fixed gh-285 the validation rule error, by removing the namespace
- Fixed the email helper class to match the custom global field value.
- Removed the spinner from export data option in the language translation area.
- Fixed the dynamic dashboard to be able to check if the view was found in the component.
- Updated the readme to reflect the new terms of gaining access to demo content.
- Added the email field type to resolve gh-283.
- Fixed the CSS inclusion error to resolve gh-278.
- Added the option to add sql_uninstall dump value to a component.
- Did some class formatting on the compiler.

### Testing Instructions
Validation rule use
Email helper fields should work
Use the new email field
Add CSS to the listview via admin view
Add custom sql uninstall dump values
JCB works stable with import and export and compiling of all components.

### Expected result
That all works correctly with no errors.

### Actual result
Fixed a few bugs and added some new features.
